activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -34,9 +34,8 @@ def test_dump_schema_information_outputs_lexically_reverse_ordered_versions_rega
     ('20100301010101'),
     ('20100201010101'),
     ('20100101010101');
-
     STR
-    assert_equal expected, schema_info
+    assert_equal expected.strip, schema_info
```

Comment:
I think it would be better to have an exact assertion here. Otherwise we are not preventing regressions.

```suggestion
    assert_equal expected, schema_info
```

Reply:
Sorry, I don't think that change makes sense in this context. There should be _no_ newlines following the expected output and calling `strip` ensures that.

In this case, it also protects against regressions because it is stripping the expected output (not the value returned by the `dump_schema_information` method). So if there is a regression in the code, this test will still catch it.

Reply:
Ahhhh yeah you're right. Sorry, my bad.

Reply:
no worries 😄